client/payment-methods/index.js

```diff
@@ -67,7 +67,7 @@ const PaymentMethods = () => {
 
 	const enabledMethods = enabledMethodIds.map( ( methodId ) =>
 		availableMethods.find( ( method ) => method.id === methodId )
-	);
+	).filter( ( method ) => method !== undefined );
```

Comment:
This was added because `woocommerce_digital_wallets` gateway is not part of the `availableMethods` list. Since we are moving towards UPE, the direction of making digital wallets as a payment gateway has changed. We no longer need to make each individual payment method a gateway. We can probably save its settings directly in the store; we also don't need digital wallets take advantage of core's [payment gateway class](https://github.com/Automattic/woocommerce/blob/trunk/includes/abstracts/abstract-wc-payment-gateway.php).